relay-server/src/actors/spooler/mod.rs

```diff
@@ -443,12 +482,10 @@ impl OnDisk {
             // 2. Make sure that if we panic and deleted envelopes cannot be read out fully, we do not lose all of them,
             // but only one batch, and the rest of them will stay on disk for the next iteration
             // to pick up.
-            //
-            // Right now we use 100 for batch size.
-            let batch_size = 100;
-            let mut envelopes = sql::delete_and_fetch(key, batch_size)
-                .fetch(&self.db)
-                .peekable();
+            let mut envelopes =
+                sql::delete_and_fetch(key, BATCH_SIZE.min(self.buffer_guard.available() as i64))
```

Comment:
Maybe move `BATCH_SIZE.min(self.buffer_guard.available() as i64)` into a separate method, it's used twice and then we can also give it a short doc string

Reply:
done in [`b2b35f2`](https://github.com/getsentry/relay/pull/2949/commits/b2b35f2b50f796378f0b84da90c7061eaa9474b6)